python/packages/core/agent_framework/_workflows/_magentic.py

```diff
@@ -1055,14 +1057,14 @@ def snapshot_state(self) -> dict[str, Any]:
             state["magentic_context"] = self._context.to_dict()
         if self._task_ledger is not None:
             state["task_ledger"] = _message_to_payload(self._task_ledger)
-        manager_state: dict[str, Any] | None = None
+
         with contextlib.suppress(Exception):
-            manager_state = self._manager.snapshot_state()
-        if manager_state:
-            state["manager_state"] = manager_state
+            state["manager_state"] = self._manager.on_checkpoint_save()
```

Comment:
I see that the exception suppression is not new, but since we're here updating code around it: should we at least log the exception at a warning level rather than completely suppressing it? Seems like an issue we don't want to simply look past if we have an error.